Xamarin.Forms.Platform.Android/AppCompat/NavigationPageRenderer.cs

```diff
@@ -604,7 +605,6 @@ void RemovePage(Page page)
 
 			// Go ahead and take care of the fragment bookkeeping for the page being removed
 			FragmentTransaction transaction = FragmentManager.BeginTransaction();
-			transaction.DisallowAddToBackStack();
```

Comment:
Don't think we need this anymore. See #919.

Reply:
You are correct - we don't. Feel free to drop it from this PR. Or leave it in and we can address them all at once.

Reply:
let's leave it in.